pom.xml

```diff
@@ -205,6 +217,18 @@
         <version>4.13.2</version>
         <scope>test</scope>
       </dependency>
+      <!-- https://mvnrepository.com/artifact/com.google.crypto.tink/tink-android -->
```

Comment:
It's not. I just keep the dependency management section around for multiple project portion which shouldn't matter here since we are never creating another module for this specific library.